src/journal/ObjectRecorder.cc

```diff
@@ -412,17 +412,18 @@ void ObjectRecorder::send_appends_aio() {
     client::guard_append(&op, m_soft_max_size);
```

Comment:
Nit: it might be nice to combine the ```guard_append``` logic directly into ```append``` so that you only need to call ```guard_append``` if the min OSD release < octopus.